client/my-sites/plans-features-main/index.tsx

```diff
@@ -834,9 +841,9 @@ const PlansFeaturesMain = ( {
 										stickyRowOffset={ masterbarHeight }
 										useCheckPlanAvailabilityForPurchase={ useCheckPlanAvailabilityForPurchase }
 										useActionCallback={ useActionCallback }
-										enableFeatureTooltips={ ! trailMapExperiment.result }
-										renderCategorisedFeatures={ trailMapExperiment.result }
-										featureGroupMap={ trailMapExperiment.result ? featureGroupMap : undefined }
+										enableFeatureTooltips={ ! isTrailMapCopy }
```

Comment:
Hmmm actually this probably should be `isTrailMapStructure`. @aneeshd16 could you help me confirm?

Reply:
Tooltips should be enabled only if we are using the old copy since the new copy does not have tooltips. So this should be `enableFeatureTooltips={ ! isTrailMapCopy }`.